src/components/ContinuationIn/SummaryBusinessHomeJurisdiction.vue

```diff
@@ -0,0 +1,341 @@
+            >
+              <div id="continuation-authorization">
+                {{ continuationAuthorization || '[Unknown]' }}
+              </div>
+            </v-col>
+          </v-row>
+        </article>
+
+        <!-- Authority Name -->
+        <article class="section-container">
+          <v-row no-gutters>
+            <v-col
+              cols="12"
+              sm="3"
+              class="pr-4"
+            >
+              <label>Authority Name</label>
+            </v-col>
+            <v-col
+              cols="12"
+              sm="9"
+              class="pt-4 pt-sm-0"
+            >
+              <div id="authority-name">
+                {{ authorityName || '[Unknown]' }}
+              </div>
+            </v-col>
+          </v-row>
+        </article>
+
+        <!-- Authorization Date -->
+        <article class="section-container">
+          <v-row no-gutters>
+            <v-col
+              cols="12"
+              sm="3"
+              class="pr-4"
+            >
+              <label>Authorization Date</label>
+            </v-col>
+            <v-col
+              cols="12"
+              sm="9"
+              class="pt-4 pt-sm-0"
+            >
+              <div id="authorization-date">
+                {{ authorizationDate || '[Unknown]' }}
+              </div>
+            </v-col>
+          </v-row>
+        </article>
+
+        <!-- Expiry Date -->
+        <article class="section-container">
+          <v-row no-gutters>
+            <v-col
+              cols="12"
+              sm="3"
+              class="pr-4"
+            >
+              <label>Expiry Date</label>
+            </v-col>
+            <v-col
+              cols="12"
+              sm="9"
+              class="pt-4 pt-sm-0"
+            >
+              <div id="expiry-date">
+                {{ expiryDate || '[Unknown]' }}
+              </div>
+            </v-col>
+          </v-row>
+        </article>
+      </div>
+    </section>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+import { Getter } from 'pinia-class'
+import { getName } from 'country-list'
+import { useStore } from '@/store/store'
+import { RouteNames } from '@/enums'
+
+@Component({})
+export default class SummaryDefineCompany extends Vue {
+  // for template
+  readonly RouteNames = RouteNames
+
+  // Getters
+  @Getter(useStore) getContinuationInBusinessInfo!: any
+  @Getter(useStore) isContinuationInStep1Valid!: boolean
+
+  get homeJurisdiction (): string {
+    const hj = this.getContinuationInBusinessInfo?.homeJurisdiction
+    if (hj?.country) {
+      const country = getName(hj.country)
+      const region = (hj.region === 'FEDERAL' ? 'Federal' : hj.region)
+      if (region) return `${region}, ${country}`
+      return country
+    }
+    return null
+  }
+
+  get identifyingNumber (): string {
+    return this.getContinuationInBusinessInfo?.homeIdentifier
```

Comment:
We don't prompt for this yet. I'm trying to sort it out with Yui and it will be handled in a future ticket.